src/Maui/Features/Creatures/Shared/CreatureDetail/AttributeScore.razor

```diff
@@ -1,15 +1,14 @@
 ﻿@using DMPowerTools.DataModels;
 
-<MudItem lg="6" md="3" Class="d-flex align-center justify-center">
-    <MudText Typo="Typo.h5">@Name</MudText>
-</MudItem>
-<MudItem lg="6" md="3">
+<MudItem lg="1" md="1" xs="1">
```

Comment:
If they’re all the same I think you can just specify 1 of them.

Reply:
Well well well. I guess that makes sense.